ccdproc/image_collection.py

```diff
@@ -69,6 +69,7 @@ def __init__(self, location=None, keywords=None, info_file=None,
         self._location = location
         self._filenames = filenames
         self._files = []
+        self._info_file = info_file
```

Comment:
Maybe wait for someone else to chime in. I actually haven't used the `info_file` ever so I'm not sure. 😄